json-endpoints/class.wpcom-json-api-get-comments-tree-endpoint.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+class WPCOM_JSON_API_Get_Comments_Tree_Endpoint extends WPCOM_JSON_API_Endpoint {
+	/**
+	 * Retrieves a list of comment data for a given site.
+	 *
+	 * @param   {string} $status   filter by status: 'all' | 'approved' | 'pending' | 'spam' | 'trash'
+	 * @param   {int}    $start_at first comment to search from going back in time
```

Comment:
Those brackets around string and int should be removed since those are for JSDocs not PHPDocs.